sunpy/net/attr.py

```diff
@@ -1,11 +1,11 @@
 # -*- coding: utf-8 -*-
 """
-Allow representation of queries as logic expressions. This module makes
-sure that attributes that are combined using the two logic operations AND (&)
-and OR (|) always are in disjunctive normal form, that is, there are only two
-levels ­- the first being disjunction and the second being conjunction. In other
-words, every combinations of attributes looks like this:
-(a AND b AND c) OR (d AND e).
+Allow representation of queries as logic expressions. This module makes sure
+that attributes that are combined using the two logic operations AND (&) and OR
+(|) always are in disjunctive normal form, that is, there are only two levels
+­- the first being disjunction and the second being conjunction. In other
+words, every combinations of attributes looks like this: (a AND b AND c) OR (d
+AND e).
```

Comment:
```suggestion
words, every combinations of attributes looks like this:

(a AND b AND c) OR (d AND e).
```
Insert a separating line so that the logical expression doesn't get wrapped